### PR TITLE
Sync release assets on a new tag

### PR DESCRIPTION
**Pull request by hellozee**

### CODE REVIEW

File: tetrateci/upload_upstream_release.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+echo "Creating a temporary directory to download $TAG release assets"
+mkdir release-tmp
```
Comment (deva26) [inline tetrateci/upload_upstream_release.sh:7]: Please create this under /tmp

Comment (hellozee) [reply tetrateci/upload_upstream_release.sh:7]: ok

File: tetrateci/upload_upstream_release.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    echo "Downloading from $url"
+    wget $url
+done
+
+PACKAGES=$(ls | grep "istio")
+
+for package in $PACKAGES; do
+    echo "Publishing $package"
+    rm -f /tmp/curl.out
+    curl -T ./$package -u$BINTRAY_USER:$API_KEY $BINTRAY_API/$TAG/$package -o /tmp/curl.out
```
Comment (deva26) [inline tetrateci/upload_upstream_release.sh:23]: We need to cross check if the package is already pushed to bintray. If so, don't push it again.

Comment (hellozee) [reply tetrateci/upload_upstream_release.sh:23]: This is an entirely new tag, do we still need to check for that?

Comment (deva26) [inline tetrateci/upload_upstream_release.sh:23]: We need to rename the file as if it is a istio flavor. That is ${TAG} to change to ${TAG}-istio-v0 in the file name.

For example, if the archive name is istio-1.8.3-linux-arm64.tar.gz, then rename it to istio-1.8.3-istio-v0-linux-arm64.tar.gz

Comment (deva26) [inline tetrateci/upload_upstream_release.sh:23]: $TAG var used in the curl URL has to be ${TAG}-istio-v0

Comment (deva26) [reply tetrateci/upload_upstream_release.sh:43]: Can we add that check? Am sure we would delete the tag and re-run manually. We should not be touching bintray if it's already released.

File: tetrateci/upload_upstream_release.sh [hunk 3]
```diff
@@ -15,8 +15,28 @@ for url in $urls; do
...
+
+for pkg in $istioctlpkgs; do
+    name=$(sed "s/istioctl-$TAG/istioctl-$TAG-istio-v0/g" <<< $pkg)
+    echo "Renaming $pkg to $name"
+    mv $pkg $name
+done
+
 PACKAGES=$(ls | grep "istio")
 
+TAG=$TAG-istio-v0
```
Comment (deva26) [inline tetrateci/upload_upstream_release.sh:38]: Can this be changed to NEWTAG=$TAG-istio-v0

Comment (deva26) [reply tetrateci/upload_upstream_release.sh:38]: And change the curl to use NEWTAG

File: tetrateci/upload_upstream_release.sh (hunk 2 above)
Comment (hellozee) [reply tetrateci/upload_upstream_release.sh:43]: Ahh okay

**Review by deva26 — APPROVED** [review]

**Pull request merged**